share/tours/inspector.tour

```diff
@@ -26,7 +26,7 @@
   <Message>
     <Text>If you have modified a value, you can click the "Reset" button to reset it to the default.
     If the value is controlled by a style setting, you can click the "Set as style" button to set the current value as the new default for the style.
-    You can also set style defaults using Format > Style.</Text>
+    You can also set style defaults using Format > Style….</Text>
```

Comment:
`>` is problematic in XML.

Matching Unicode use of the ellipsis, I’d use →, that is, `Format → Style…` here (and everywhere).

Reply:
It work here though. And it has been agreed on to use > rather than → everywhere after a longer discussion and had then been cast in concrete via https://musescore.org/en/handbook/developers-handbook/text-style-guide